packages/react-scripts/README.md

```diff
@@ -1,7 +1,92 @@
-# react-scripts
+# backpack-react-scripts
 
-This package includes scripts and configuration used by [Create React App](https://github.com/facebook/create-react-app).<br>
-Please refer to its documentation:
+### **Important:** The currently supported version of **CRA** by `backpack-react-scripts` is up to `v3.4.1`. Versions above this will not work.
```

Comment:
This is no longer true, right? we should support all versions of `create-react-app`

Reply:
This is true as this is what happened before we upgraded to CRA 3+ where templating was introducted and caused this script to fail. You can try it now with a BRS v7 version

Reply:
The plan would be to bump it as we keep it up to date with CRA